### PR TITLE
Chado tools

### PR DESCRIPTION
**Pull request by puethe**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

command line tool to access CHADO databases

### CODE REVIEW

File: recipes/chado-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sanger-pathogens/{{ name }}/archive/v{{ version }}.{{ file_ext }}
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  number: 0
+  noarch: generic
```
Comment (bgruening) [inline recipes/chado-tools/meta.yaml]: I guess this can be `noarch: python`?

Comment (puethe) [reply recipes/chado-tools/meta.yaml]: I'm not sure. It's a command line tool, and according to [the documentation](https://bioconda.github.io/linting.html#should-be-noarch), it should then be "generic" - if I understood correctly.

Comment (puethe) [reply recipes/chado-tools/meta.yaml]: Tests on my machine pass flawlessly with either setting. Let me know what you think is better.

Comment (bgruening) [reply recipes/chado-tools/meta.yaml]: python is preferred I guess.

Comment (puethe) [reply recipes/chado-tools/meta.yaml]: Ok, thanks. Changed it.

**Pull request merged**